### PR TITLE
update jenkins doc for the new jenkins-agent-* images and deprecation…

### PR DESCRIPTION
**Pull request by gabemontero**
… of jenkins-slave-maven* and jenkins-slave-nodejs* images

@openshift/sig-developer-experience @bmcelvee ptal

For https://trello.com/c/YQm7osws/1493-5-jenkins-slave-scl-dep-updatesjenkinsintegration

### CODE REVIEW

**Comment by gabemontero** [comment]
[(5) Jenkins slave SCL dep updates[jenkins_integration]](https://trello.com/c/YQm7osws/1493-5-jenkins-slave-scl-dep-updatesjenkinsintegration)

**Comment by gabemontero** [comment]
[(5) Jenkins slave SCL dep updates[jenkins_integration]](https://trello.com/c/YQm7osws/1493-5-jenkins-slave-scl-dep-updatesjenkinsintegration)

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -391,23 +391,28 @@ you can use in deployment configuration instead of the official Jenkins image.
...
-xref:jenkins_slaves.adoc#overview[Jenkins Slaves] for more information.
+To use the Kubernetes plug-in, {product-title} provides five images suitable
+for use as Jenkins agents: the *_Base_*, *_Maven_*, and *_Node.js_* images. See
+xref:jenkins_agents.adoc#overview[Jenkins Agents] for more information.
 
-Both the Maven and Node.js slave images are automatically configured as
+NOTE:  the jenkins-slave-maven-* and jenkins-slave-nodejs-* images are being
+marked as deprecated during the v3.10 release cycle.  The images will still
+exist in the interim so users can migrate their applications to the newer
+jenkins-agent-maven-34-* and jenkins-agent-nodejs-8-* images.
```
Comment (bparees) [inline using_images/other_images/jenkins.adoc]: s/34/35/

Comment (bparees) [reply using_images/other_images/jenkins.adoc]: or better, just "jenkins-agent-maven-* and jenkins-agent-nodejs-* images"

File: using_images/other_images/jenkins_slaves.adoc [hunk 2]
```diff
@@ -13,33 +13,41 @@ toc::[]
...
 == Overview
-{product-title} provides three images suitable for use as Jenkins slaves: the
+{product-title} provides three images suitable for use as Jenkins agents: the
 *_Base_*, *_Maven_*, and *_Node.js_* images.
 
-The first is a link:https://github.com/openshift/jenkins/tree/master/slave-base[base image] for Jenkins slaves:
+NOTE:  the jenkins-slave-maven-* and jenkins-slave-nodejs-* images are being
+marked as deprecated during the v3.10 release cycle.  The images will still
+exist in the interim so users can migrate their applications to the newer
+jenkins-agent-maven-34-* and jenkins-agent-nodejs-8-* images.
```
Comment (bparees) [inline using_images/other_images/jenkins_slaves.adoc]: "jenkins-agent-maven-* and jenkins-agent-nodejs-* images"

**Review by bparees — CHANGES_REQUESTED** [review]
nits

**Comment by gabemontero** [comment]
update for nits pushed @bparees thanks

**Comment by bparees** [comment]
lgtm

**Comment by gabemontero** [comment]
commits squashed

**Comment by bparees** [comment]
@openshift/team-documentation ptal

**Comment by vikram-redhat** [comment]
@bmcelvee - PTAL.

**Comment by bmcelvee** [comment]
Thanks, @gabemontero! LGTM. This is for 3.10, right?

@vikram-redhat, we don't yet have a 3.10 label or branch. Should I hold off on merging until those are available? Thanks!

**Comment by gabemontero** [comment]
YW @bmcelvee 

And yes, this is for 3.10

**Comment by vikram-redhat** [comment]
@bmcelvee - yes, sorry no 3.10 branch yet. I have created the 3.10 label though and attached it. 

@gabemontero or @bparees - should it go into the master so it's available upstream in the meantime?

**Comment by bparees** [comment]
> @gabemontero or @bparees - should it go into the master so it's available upstream in the meantime?

as long as there's no risk it ends up in the v3.9 product docs, yes please.

**Pull request merged**